wagtail/admin/templates/wagtailadmin/bulk_actions/confirmation/go_back.html

```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+<a href="{{ next }}" class="button button-secondary">{% trans "Go back" %}</a>
```

Comment:
This feels pretty slight for its own template - maybe move to the base one?

Reply:
I am not sure how that will work, since this is conditionally rendered, and the condition will be different for pages, documents and images. Defining it in the base template would mean we would have to overwrite it every time using 
`{{ block go_back }}{{ endblock }}` if we have to not display it.